public/app/features/profile/UserProfileEditPage.tsx

```diff
@@ -13,6 +18,9 @@ import UserSessions from './UserSessions';
 import { UserTeams } from './UserTeams';
 import { changeUserOrg, initUserProfilePage, revokeUserSession, updateUserProfile } from './state/actions';
 
+const TAB_QUERY_PARAM = 'tab';
+const CORE_SETTINGS_TAB = 'Core';
```

Comment:
think we should make this lower case
```suggestion
const CORE_SETTINGS_TAB = 'core';
```